webapp/src/containers/SideBar.js

```diff
@@ -66,7 +66,7 @@ const SideBar = ({ user, onLogout, triggerSideBarPosition }) => {
       <Box className={classes.optionLink}>
         <LoginModal isSideBar />
       </Box>
-      <Box className={classes.optionLink}>
+      <Box className={classes.optionLink} onClick={() => triggerSideBarPosition()}>
```

Comment:
What if we use `onClick={triggerSideBarPosition}`, since that function calling doesn't have any parameter